packages/rum/src/apm-base.js

```diff
@@ -85,7 +82,12 @@ export default class ApmBase {
         }
       } else {
         this._disable = true
-        loggingService.info('RUM agent is inactive')
+        /**
+         * Creation of LoggingService should happen only after config update.
+         * This order gurantees that we use the proper logLevel
+         */
+        const loggingService = this.serviceFactory.getService(LOGGING_SERVICE)
+        loggingService.warn('RUM agent is inactive')
```

Comment:
moved from `info` to `warn` as our default level is warn.

Reply:
`warn` makes more sense in this case anyway :+1: